composer.json

```diff
@@ -19,16 +19,16 @@
     "require": {
         "php": "^7.4|^8.0",
         "ext-json": "*",
-        "composer-plugin-api": "^1.0|^2.0",
+        "composer-plugin-api": "^2.0",
         "jfcherng/php-sequence-matcher": "^3.2",
         "yiisoft/var-dumper": "^1.1"
     },
     "require-dev": {
+        "composer/composer": "^2.0",
```

Comment:
Why require `composer/composer` ?

Reply:
For tests (`require-dev`).

Reply:
The plugin uses some classes from it, for example `Composer\Util\Filesystem`.

Reply:
But before that, it worked without this requirements.

Reply:
I removed the version, look carefully at the diff of changes.

Reply:
Sorry, I didn't notice